<!DOCTYPE html []>
<html>
  <head>
    <meta charset="UTF-8" />
    <meta name="author" content="MarkdownViewer++" />
    <title>readme.txt</title>
    <style type="text/css">
            
/* Avoid page breaks inside the most common attributes, especially for exports (i.e. PDF) */
td, h1, h2, h3, h4, h5, p, ul, ol, li {
    page-break-inside: avoid; 
}
html {
  padding: 1in;
  font-family: arial;
}
a {
  color: black;
  text-decoration: none;
}
        </style>
  </head>
  <body>
    <h1 id="readme">Readme</h1>
    <h2 id="coronavirus-macroevolution">Coronavirus Macroevolution</h2>
    <h3 id="sars-cov-2-genomes-are-evolving-gradually">SARS-CoV-2 Genomes Are Evolving Gradually</h3>
    <ul>
      <li>We first aim to describe the mode of SARS-CoV-2 evolution and how it compares
to its inferred slow tempo so far (Bedford et al. 2020; Jia et al. 2020)?
<ul><li>To start, we focus on SARS-CoV-2 whole genomes.
<ul><li>Once they jumped into and spread among humans, has SARS-CoV-2 been
evolving in a punctuational manner, so that a large proportion of
their genomic divergences occurred during transmission events?</li><li>The null hypothesis is that they have been evolving gradually.</li><li>To test for punctuated evolution (see Webster et al. 2003; Pagel et
al. 2006), we use phylogenetic generalized least squares (PGLS) to
regress the total phylogenetic path lengths of the genomes (distances
from the root of a phylogenetic tree to the tips) on the net number
of transmissions (nodes in a tree).</li><li>We expect the net number of transmissions along SARS-CoV-2 lineages
to correlate with (slope &gt; 0) and explain the variation in
accumulated mutations (<em>R<sup>2</sup></em> &gt; 0), controlling for shared ancestry.</li><li>First, we downloaded a molecular tree of SARS-CoV-2 (<em>n</em> = 3,944
genomes) from <code>Nextstrain</code> (Sagulenko et al. 2017; Hadfield et al.
2018; <a href="https://nextstrain.org/ncov">https://nextstrain.org/ncov</a>).
<ul><li>Downloaded on: 4/15/2020, 9:56:14 AM</li><li>The genomes come from various authors and labs, deposited at
<code>GISAID</code> (Shu and McCauley 2017; <a href="https://www.gisaid.org/">https://www.gisaid.org/</a>).</li><li>Bioinformatic notes (see <a href="https://github.com/nextstrain/ncov">https://github.com/nextstrain/ncov</a>):
<ul><li>Site numbering and genome structure use Wuhan-Hu-1/2019
(GenBank Accession number: MN908947; Wu et al. 2020) as the
reference.</li><li>The phylogeny is rooted relative to early samples from Wuhan.</li><li>There were single nucleotide polymorphisms (SNPs) present in
the nCoV samples in the first and last few bases of the
alignment that were masked as likely sequencing artifacts.</li></ul></li><li>How does the <code>Nextstrain</code> team model the rate of molecular
evolution (e.g., substitution model, across-site rate variation,
clock model, and heterotachy)?</li><li>The branch lengths are in the unit of the expected number of
substitutions per site.</li><li>Once our analytical pipeline is up and running, we can redo the
analyses using the most up-to-date tree.</li></ul></li><li>Using <code>FigTree</code> 1.4.4 (Rambaut 2017), we saved the tree in a Newick
format with the tip labels enclosed within single quotation marks.
<ul><li>This tip label edit is to prevent downstream errors.</li></ul></li><li>Using the <code>R</code> 3.6.2 (R Core Team 2019) package <code>phytools</code> 0.6.99
(Revell 2012), we converted the Newick tree into a NEXUS format.</li><li>Using the package <code>ape</code> 5.3 (Paradis and Schliep 2019), we extracted
the total path lengths (diagonals of the variance-covariance matrix)
and the number of nodes along the paths.</li><li>Using the regex functionality in <code>Notepad++</code> 7.5.8, we fixed the
codings of clades with 'anagenetic' taxa in the tree file.
<ul><li>Example:
<ul><li>Problematic: <code>(10:0,(11:0):5);</code></li><li>Correct: <code>(10:0,11:5);</code></li><li>Regex (search): <code>\(([0-9][0-9]):0\)</code><ul><li>The backslashes escape the parentheses in front of them.</li><li>The unescaped parentheses set the digits they enclose as
a group.</li></ul></li><li>Regex (replace): <code>\1</code><ul><li>The replacement is the group designated above.</li></ul></li></ul></li></ul></li><li>Using the maximum likelihood (ML) algorithm in <code>BayesTraits</code> 3.0.1
(Pagel 1999; <a href="http://www.evolution.rdg.ac.uk/BayesTraitsV3.0.1/BayesTraitsV3.0.1.html">http://www.evolution.rdg.ac.uk/BayesTraitsV3.0.1/BayesTraitsV3.0.1.html</a>),
we estimated the parameters of the regression model above.
<ul><li>PGLS accounts for the shared inheritance of branch lengths.</li><li>Our PGLS model assumes that the tree is correct.</li><li>We fixed Pagel's <em>λ</em> (Pagel 1999), a measure of phylogenetic
signal, to one.</li><li>We used default ML search settings.</li><li>We executed two ML runs and compared the results.</li></ul></li><li>There is very little evidence to suggest that SARS-CoV-2 genomes
evolved in a punctuated manner (slope = 0.0084 ± 0.0032;
<em>R<sup>2</sup></em> = 0.0018).
<ul><li>The two ML runs produced practically identical results.</li></ul></li><li>SARS-CoV-2 genomes are most likely evolving gradually, with much of
the mutations occurring in between net transmission events, as
depicted in the tree.</li><li>We, therefore, find that the tempo and mode of SARS-CoV-2 evolution
are linked.</li><li>We created and saved scatter plots of the regression using the <code>R</code>
packages <code>Cairo</code> 1.5.10 (Urbanek and Horner 2019), <code>ggplot2</code> 3.2.1
(Wickham 2009), <code>ggthemes</code> 4.2.0 (Arnold 2019), <code>htmlwidgets</code> 1.5.1
(Vaidyanathan et al. 2019), <code>plotly</code> 4.9.1 (Sievert 2018), and
<code>svglite</code> 1.2.2 (Wickham et al. 2019).
<ul><li>We did not observe any potential outliers.</li></ul></li><li>We ran regression diagnostics by plotting the residuals against
fitted values, and a histogram of the residuals using the packages
<code>ggExtra</code> 0.9 (Attali and Baker 2019), <code>ggplot2</code>, <code>ggthemes</code>, and
<code>svglite</code>.
<ul><li>The residuals vs. fitted values plot does not indicate a
violation of residual homogeneity.</li><li>The histogram shows that the residuals are normally distributed.</li></ul></li><li>We checked whether an artifact of phylogeny reconstruction, the
node-density effect (Webster et al. 2003; Venditti et al. 2006),
biases our punctuation test.
<ul><li>The node-density effect is the underestimation of branch lengths
in tree regions with fewer taxa.</li><li>We executed the <em>δ</em> test, which tests for a curvilinear
relationship between path length (<em>x</em>) and the number of nodes
(<em>n</em>).
<ul><li>Equation: <em>x</em> = <em>β</em>·<em>n</em><sup>1/<em>δ</em></sup><ul><li><em>β:</em> rate of change between path length and the number of
nodes</li></ul></li><li>We expect <em>δ</em> &gt; 1 when the node-density effect is present.</li></ul></li><li>We used <code>R</code> to estimate <em>δ</em> by fitting a log-log regression.</li></ul></li><li>The <em>δ</em> test suggests that the node-density effect is present
(<em>δ</em> = 8.17), but the <em>P</em> value of the exponent term is high
(<em>P</em> = 0.51), and the <em>R<sup>2</sup></em> of the regression is low
(<em>R<sup>2</sup></em> = 0.00011).</li><li>Non-random sampling regarding time and location can affect how we
interpret our results.
<ul><li>One way to roughly assess such sampling bias is to randomly drop
~10% of the genomes in the tree, redo the analysis, and repeat
100 times.</li></ul></li></ul></li><li>A follow-up question is whether only certain parts of the tree exhibit
signatures of punctuated evolution.</li><li>Another potential follow-up is to test for punctuated evolution for each
gene in the genome.</li></ul></li>
    </ul>
    <h2 id="brief-literature-review">Brief Literature Review</h2>
    <ul>
      <li>
        <strong>Holmes (2003)</strong>
        <br />
summarized the general structure of SARS-associated coronavirus virion</li>
      <li>
        <strong>Gorbalenya et al. (2020)</strong>
        <br />
named SARS-CoV-2</li>
      <li>
        <strong>van Doremalen et al. (2020)</strong>
evaluated the stability of SARS-CoV-2 and SARS-CoV-1 in aerosols and on
various surfaces, and estimated their decay rates</li>
      <li>
        <strong>Wang et al. (2020)</strong>
        <br />
summarized all three major coronaviruses outbreaks</li>
      <li>
        <strong>Wu et al. (2020a)</strong>
        <br />
analyzed the SARS-CoV-2 genome from one of the likely patient zeroes</li>
      <li>
        <strong>Wu et al. (2020b)</strong>
        <br />
showed that SARS-CoV-2 viral RNA could persists in fecal samples longer
than in respiratory ones</li>
      <li>
        <strong>Wu et al. (2020c)</strong>
        <br />
championed wastewater surveillance to help estimate the prevalence of
SARS-CoV-2 in a population</li>
      <li>
        <strong>Xiao et al. (2020)</strong>
        <br />
hypothesized that SARS-CoV-2 originated from the recombination of pangolin
and bat CoV-like viruses</li>
      <li>
        <strong>Xu et al. (2020)</strong>
        <br />
briefly reviewed the origin and transmission of SARS-CoV-2</li>
      <li>
        <strong>Zhang et al. (in press):</strong>
        <br />
proposed pangolin as a possible natural reservoir of SARS-CoV-2</li>
      <li>
        <strong>Zhou et al. (2020):</strong>
        <br />
analyzed the SARS-CoV-2 genomes from five of the likely patient zeroes</li>
    </ul>
    <h2 id="references">References</h2>
    <ul>
      <li>Attali D., Baker C. 2019. ggExtra: Add marginal histograms to "ggplot2", and
more "ggplot2" enhancements. R package.</li>
      <li>Arnold J.B. 2019. ggthemes: Extra themes, scales and geoms for "ggplot2." R
package.</li>
      <li>Bedford T., Greninger A.L., Roychoudhury P., Starita L.M., Famulare M., Huang
M.-L., Nalla A., Pepper G., Reinhardt A., Xie H., Shrestha L., Nguyen T.N.,
Adler A., Brandstetter E., Cho S., Giroux D., Han P.D., Fay K., Frazar
C.D., Ilcisin M., Lacombe K., Lee J., Kiavand A., Richardson M., Sibley
T.R., Truong M., Wolf C.R., Nickerson D.A., Rieder M.J., Englund J.A.,
The Seattle Flu Study Investigators, Hadfield J., Hodcroft E.B., Huddleston
J., Moncla L.H., Müller N.F., Neher R.A., Deng X., Gu W., Federman S., Chiu
C., Duchin J., Gautom R., Melly G., Hiatt B., Dykema P., Lindquist S.,
Queen K., Tao Y., Uehara A., Tong S., MacCannell D., Armstrong G.L., Baird
G.S., Chu H.Y., Jerome K.R. 2020. <em><strong>Cryptic transmission of SARS-CoV-2 in
Washington State</strong></em>. medRxiv.:2020.04.02.20051417.</li>
      <li>Gorbalenya A.E., Baker S.C., Baric R.S., de Groot R.J., Drosten C., Gulyaeva
A.A., Haagmans B.L., Lauber C., Leontovich A.M., Neuman B.W., Penzar D.,
Perlman S., Poon L.L.M., Samborskiy D.V., Sidorov I.A., Sola I., Ziebuhr
J., Coronaviridae Study Group of the International Committee on Taxonomy of
Viruses. 2020. <em><strong>The species <em>Severe acute respiratory syndrome-related
coronavirus</em>: Classifying 2019-nCoV and naming it SARS-CoV-2</strong></em>. Nat.
Microbiol. 1–9.</li>
      <li>Hadfield J., Megill C., Bell S.M., Huddleston J., Potter B., Callender C.,
Sagulenko P., Bedford T., Neher R.A. 2018. Nextstrain: Real-time tracking
of pathogen evolution. Bioinformatics. 34:4121–4123.</li>
      <li>Holmes K.V. 2003. <em><strong>SARS-associated coronavirus</strong></em>. N. Engl. J. Med.
348:1948–1951.</li>
      <li>Jia Y., Shen G., Zhang Y., Huang K.-S., Ho H.-Y., Hor W.-S., Yang C.-H., Li
C., Wang W.-L. 2020. <em><strong>Analysis of the mutation dynamics of SARS-CoV-2
reveals the spread history and emergence of RBD mutant with lower ACE2
binding affinity</strong></em>. bioRxiv.:2020.04.09.034942.</li>
      <li>Pagel M. 1999. Inferring the historical patterns of biological evolution.
Nature. 401:877–884.</li>
      <li>Pagel M., Venditti C., Meade A. 2006. Large punctuational contribution of
speciation to evolutionary divergence at the molecular level. Science.
314:119–121.</li>
      <li>Paradis E., Schliep K. 2019. ape 5.0: An environment for modern phylogenetics
and evolutionary analyses in R. Bioinformatics. 35:526–528.</li>
      <li>R Core Team. 2019. R: A language and environment for statistical computing.
Vienna, Austria: R Foundation for Statistical Computing.</li>
      <li>Rambaut A. 2017. FigTree-version 1.4.4, a graphical viewer of phylogenetic
trees.</li>
      <li>Revell L.J. 2012. phytools: An R package for phylogenetic comparative biology
(and other things). Methods Ecol. Evol. 3:217–223.</li>
      <li>Sagulenko P., Puller V., Neher R.A. 2017. TreeTime: Maximum-likelihood
phylodynamic analysis. Virus Evol. 4.</li>
      <li>Sievert C. 2018. plotly for R.</li>
      <li>Shu Y., McCauley J. 2017. GISAID: Global initiative on sharing all influenza
data – from vision to reality. Euro Surveill. 22.</li>
      <li>Urbanek S., Horner J. 2019. Cairo: R graphics device using cairo graphics
library for creating high-quality bitmap (PNG, JPEG, TIFF), vector (PDF,
SVG, PostScript) and display (X11 and Win32) output. R package.</li>
      <li>van Doremalen N., Bushmaker T., Morris D.H., Holbrook M.G., Gamble A.,
Williamson B.N., Tamin A., Harcourt J.L., Thornburg N.J., Gerber S.I.,
Lloyd-Smith J.O., de Wit E., Munster V.J. 2020. Aerosol and surface
stability of SARS-CoV-2 as compared with SARS-CoV-1. N. Engl. J. Med.</li>
      <li>Vaidyanathan R., Xie Y., Allaire J.J., Chang J., Russell K. 2019.
htmlwidgets: HTML widgets for R. R package.</li>
      <li>Venditti C., Meade A., Pagel M. 2006. Detecting the node-density artifact in
phylogeny reconstruction. Syst. Biol. 55:637–643.</li>
      <li>Wang C., Horby P.W., Hayden F.G., Gao G.F. 2020. <em><strong>A novel coronavirus
outbreak of global health concern</strong></em>. Lancet. 395:470–473.</li>
      <li>Webster A.J., Payne R.J.H., Pagel M. 2003. Molecular phylogenies link rates
of evolution and speciation. Science. 301:478–478.</li>
      <li>Wickham H. 2009. ggplot2: Elegant graphics for data analysis. New York:
Springer-Verlag.</li>
      <li>Wickham H., Henry L., Luciani T.J., Decorde M., Lise V. 2019. svglite: An
"SVG" graphics device. R package.</li>
      <li>Wu F., Zhao S., Yu B., Chen Y.-M., Wang W., Song Z.-G., Hu Y., Tao Z.-W.,
Tian J.-H., Pei Y.-Y., Yuan M.-L., Zhang Y.-L., Dai F.-H., Liu Y., Wang
Q.-M., Zheng J.-J., Xu L., Holmes E.C., Zhang Y.-Z. 2020a. <em><strong>A new
coronavirus associated with human respiratory disease in China</strong></em>. Nature.
579:265–269.</li>
      <li>Wu Y., Guo C., Tang L., Hong Z., Zhou J., Dong X., Yin H., Xiao Q., Tang Y.,
Qu X., Kuang L., Fang X., Mishra N., Lu J., Shan H., Jiang G., Huang X.
2020b. Prolonged presence of SARS-CoV-2 viral RNA in faecal samples.
Lancet Gastroenterol. Hepatol. 5:434–435.</li>
      <li>Wu F., Xiao A., Zhang J., Gu X., Lee W.L., Kauffman K., Hanage W., Matus M.,
Ghaeli N., Endo N., Duvallet C., Moniz K., Erickson T., Chai P., Thompson
J., Alm E. 2020c. SARS-CoV-2 titers in wastewater are higher than expected
from clinically confirmed cases. medRxiv.:2020.04.05.20051540.</li>
      <li>Xiao K., Zhai J., Feng Y., Zhou N., Zhang X., Zou J.-J., Li N., Guo Y., Li
X., Shen X., Zhang Z., Shu F., Huang W., Li Y., Zhang Z., Chen R.-A., Wu
Y.-J., Peng S.-M., Huang M., Xie W.-J., Cai Q.-H., Hou F.-H., Liu Y., Chen
W., Xiao L., Shen Y. 2020. <em><strong>Isolation and characterization of
2019-nCoV-like coronavirus from Malayan pangolins</strong></em>.
bioRxiv.:2020.02.17.951335.</li>
      <li>Xu Y. 2020. <em><strong>Unveiling the origin and transmission of 2019-nCoV</strong></em>. Trends
Microbiol. 28:239–240.</li>
      <li>Zhang T., Wu Q., Zhang Z. 2020. <em><strong>Probable pangolin origin of SARS-CoV-2
associated with the COVID-19 outbreak</strong></em>. Curr. Biol. 30:1346-1351.e2.</li>
      <li>Zhou P., Yang X.-L., Wang X.-G., Hu B., Zhang L., Zhang W., Si H.-R., Zhu Y.,
Li B., Huang C.-L., Chen H.-D., Chen J., Luo Y., Guo H., Jiang R.-D., Liu
M.-Q., Chen Y., Shen X.-R., Wang X., Zheng X.-S., Zhao K., Chen Q.-J.,
Deng F., Liu L.-L., Yan B., Zhan F.-X., Wang Y.-Y., Xiao G.-F., Shi Z.-L.
2020. <em><strong>A pneumonia outbreak associated with a new coronavirus of probable
bat origin</strong></em>. Nature. 579:270–273.</li>
    </ul>
    <h2 id="nature-coronavirus-collection">
      <em>Nature</em> Coronavirus Collection</h2>
    <ul>
      <li>
        <strong>
          <a href="https://www.nature.com/collections/hajgidghjb">Coronavirus</a>
        </strong>
      </li>
      <li>
        <strong>
          <a href="https://doi.org/10.1038/d41586-020-00154-w">Coronavirus latest: Global infections pass two million</a>
        </strong>
      </li>
      <li>
        <a href="https://doi.org/10.1038/d41586-020-00166-6">China coronavirus: Six questions scientists are asking</a>
      </li>
      <li>
        <a href="https://doi.org/10.1038/d41586-020-00973-x">How sewage could reveal true scale of coronavirus outbreak</a>
      </li>
      <li>
        <a href="https://doi.org/10.1038/d41586-020-00364-2">Did pangolins spread the China coronavirus to people?</a>
      </li>
      <li>
        <a href="https://doi.org/10.1038/d41586-020-00180-8">Why snakes probably aren't spreading the new China virus</a>
      </li>
      <li>
        <a href="https://doi.org/10.1038/d41586-020-00760-8">How much is coronavirus spreading under the radar?</a>
      </li>
      <li>
        <a href="https://doi.org/10.1038/d41586-020-00758-2">The coronavirus pandemic in five powerful charts</a>
      </li>
    </ul>
    <h2 id="science-coronavirus-collection">
      <em>Science</em> Coronavirus Collection</h2>
    <ul>
      <li>
        <strong>
          <a href="https://www.sciencemag.org/coronavirus-research-commentary-and-news?utm_source=newsletters&amp;utm_medium=newsletter&amp;utm_campaign=coronavirus600x190-24027&amp;et_rid=681382821&amp;et_cid=3261778#">Coronavirus: Research, Commentary, and News</a>
        </strong>
      </li>
    </ul>
    <h2 id="other-resources">Other Resources</h2>
    <ul>
      <li>
        <strong>
          <a href="https://icite.od.nih.gov/covid19/search">COVID-19 Portofolio</a>
        </strong>
      </li>
      <li>
        <strong>
          <a href="https://www.ncbi.nlm.nih.gov/research/coronavirus">LitCOVID</a>
        </strong>
      </li>
      <li>
        <strong>
          <a href="https://nextstrain.org">Nextstrain</a>
        </strong>
      </li>
      <li>
        <strong>
          <a href="https://nextstrain.org/ncov">Nextstrain: Genomic epidemiology of novel coronavirus</a>
        </strong>
      </li>
      <li>
        <strong>
          <a href="https://nextstrain.org/groups/blab/beta-cov">Nextstrain: Genetic diversity of betacoronaviruses including novel
coronavirus (nCoV)</a>
        </strong>
      </li>
      <li>
        <strong>
          <a href="https://nextstrain.org/groups/blab/sars-like-cov">Nextstrain: Phylogeny of SARS-like betacoronaviruses including novel
coronavirus SARS-CoV-2</a>
        </strong>
      </li>
      <li>
        <strong>
          <a href="https://coronavirus.jhu.edu">Johns Hopkins University &amp; Medicine Coronavirus Resource Center</a>
        </strong>
      </li>
      <li>
        <strong>
          <a href="https://github.com/CSSEGISandData/COVID-19">2019 Novel Coronavirus COVID-19 (2019-nCoV) Data Repository by Johns
Hopkins CSSE</a>
        </strong>
      </li>
      <li>
        <strong>
          <a href="https://www.newyorker.com/science/elements/from-bats-to-human-lungs-the-evolution-of-a-coronavirus#intcid=recommendations_the-new-yorker-right-rail-popular_bb51f0e8-6f2e-4099-b947-1bed209b9498_popular4-1">From Bats to Human Lungs, the Evolution of a Coronavirus</a>
        </strong>
      </li>
      <li>
        <strong>
          <a href="https://www.nytimes.com/interactive/2020/04/03/science/coronavirus-genome-bad-news-wrapped-in-protein.html">Bad News Wrapped in Protein: Inside the Coronavirus Genome</a>
        </strong>
      </li>
      <li>
        <a href="https://www.newsweek.com/coronavirus-mutation-study-covid-19-1497745">First 'Significant' Coronavirus Mutation Discovered In Preliminary Study</a>
      </li>
      <li>
        <a href="https://podcasts.apple.com/us/podcast/this-week-in-virology/id300973784?i=1000469609855">TWiV Special: Coronavirus update with Mark Denison, MD</a>
      </li>
      <li>
        <a href="http://virological.org/t/phylodynamic-analysis-176-genomes-6-mar-2020/356">Phylodynamic Analysis | 176 genomes | 6 Mar 2020</a>
      </li>
      <li>
        <a href="http://virological.org/t/ncov-2019-codon-usage-and-reservoir-not-snakes-v2/339">nCoV-2019 codon usage and reservoir (not snakes v2)</a>
      </li>
      <li>
        <a href="https://covid19.healthdata.org/united-states-of-america">COVID-19 Projections</a>
      </li>
      <li>
        <a href="https://covidactnow.org">CovidActNow</a>
      </li>
      <li>
        <a href="https://taming-the-beast.org/tutorials/Molecular-Dating-Tutorial">Molecular dating using heterochronous data and substitution model
averaging</a>
      </li>
    </ul>
  </body>
</html>
